contrib/chromium/template.py

```diff
@@ -171,19 +177,23 @@ def post_patch(self):
 
 def do_configure(self):
     # compile gn early so it can be used to generate gni stuff
-    self.do(
-        "./tools/gn/bootstrap/bootstrap.py",
-        f"-j{self.make_jobs}",
-        "--skip-generate-buildfiles",
-    )
+    with self.stamp("gn_bootstrap") as s:
```

Comment:
fwiw i don't think any of these stamps are really necessary here and actually get in the way; there is no harm in re-running them on partial configure, and it might actually be wanted

Reply:
i don't think they succeed when reran (i'll check real quick), but i agree for gn_configure that that is nice to manually retrigger..

Reply:
(they = unbundle specifically, the other two are fine for sure so removed locally)

Reply:
yeah my memory was wrong, they're all ok to rerun